coalib/bears/Bear.py

```diff
@@ -460,14 +460,17 @@ def execute(self, *args, debug=False, **kwargs):
                 self.err('Bear {} violated one-based offset convention.'
                          .format(name), str(exc))
 
-            if self.kind() == BEAR_KIND.LOCAL:
-                self.err('Bear {} failed to run on file {}. Take a look '
-                         'at debug messages (`-V`) for further '
-                         'information.'.format(name, args[0]))
+            if self.kind() == BEAR_KIND.LOCAL
```

Comment:
Ci is showing syntax error on this line So I think the way you took the and part on next line is messibg the syntax. You'll need to fix that before anything else.

Reply:
@Man-Jain right I too looked and found the same

Reply:
but actually if i am writting on a single line then problem and if i am indenting the remaining part on next line then problem what should i do ?

Reply:
Put the conditions in brackets.